### PR TITLE
fix displaing of Profile Picture

### PR DESCRIPTION
**Pull request by VitalyiKhlivnyi**
## JIRA

* [Main JIRA ticket](http://ssu-jira.softserveinc.com/browse/LEAR-22)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [x] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by tarassito — DISMISSED** [review]

File: eventually/static/src/containers/profile/ProfileEdit.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const style_header = {
 };
 
 const style_buttons = {
```
Comment (iryn34) [inline eventually/static/src/containers/profile/ProfileEdit.js]: use camelCase style for const

File: eventually/static/src/containers/profile/ProfileEdit.js [hunk 2]
```diff
@@ -43,7 +43,12 @@ const style_buttons = {
 const style_container = {
```
Comment (iryn34) [inline eventually/static/src/containers/profile/ProfileEdit.js]: use camelCase style for const

Comment (dotiurin) [inline eventually/static/src/containers/profile/ProfileEdit.js]: use camelCase

**Review by ashatna — DISMISSED** [review]

File: eventually/static/src/containers/profile/ProfileView.js [hunk 3]
```diff
@@ -32,10 +32,10 @@ const styleLowerMain2 = {
 };
 
 const style_container = {
```
Comment (dotiurin) [inline eventually/static/src/containers/profile/ProfileView.js]: use camelCase

Comment (iryn34) [inline eventually/static/src/containers/profile/ProfileView.js]: use camelCase style for const

File: eventually/static/src/containers/profile/ProfileEdit.js (hunk 1 above)
Comment (dotiurin) [inline eventually/static/src/containers/profile/ProfileEdit.js]: use camelCase

**Review by iryn34 — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by tarassito — DISMISSED** [review]

File: eventually/static/src/containers/fileUpload/FileUpload.js [hunk 4]
```diff
@@ -26,20 +26,24 @@ import {imageValidator} from './FileUploadHelper';
...
+
 export default class FileUpload extends React.Component {
 
     constructor(props){
         super(props);
         this.state = {
-            image_name: '',
-            image_src: 'https://s3.eu-west-2.amazonaws.com/eventually-photos/',
+            imageName: '',
+            imageSrc: 'https://s3.eu-west-2.amazonaws.com/eventually-photos/',
```
Comment (lhalam) [inline eventually/static/src/containers/fileUpload/FileUpload.js]: use helper.s3Root

**Review by lhalam — DISMISSED** [review]

File: eventually/static/src/containers/fileUpload/FileUpload.js (hunk 4 above)
Comment (VitalyiKhlivnyi) [reply eventually/static/src/containers/fileUpload/FileUpload.js]: Done

**Review by ashatna — APPROVED** [review]

**Pull request merged**